.eslintrc.json

```diff
@@ -76,6 +76,6 @@
         "exports": true
     },
     "extends": [
-        "eslint:recommended"
+        "eslint:recommended", "google"
```

Comment:
I'm just curious, it looks like in blockly samples we use "eslint-config-google" instead of "google". Is there a difference?

Reply:
That's a little suspicious, because this is how the readme says to do it.